pkg/lclient/images.go

```diff
@@ -21,7 +21,7 @@ func (dc *Client) PullImage(image string) error {
 	}
 	defer out.Close()
 
-	if klog.V(4) {
+	if klog.V(2) {
```

Comment:
shouldn't this be `3`?

Reply:
Ah sorry, it should be V(1) since lclient using Docker, which is an experimental feature. My mistake and thank you for catching it!